src/servers/bria-trigger.ts

```diff
@@ -30,11 +23,25 @@ const utxoDetectedEventHandler = async ({
 }: {
   event: UtxoDetected
 }): Promise<true | ApplicationError> => {
-  return Wallets.addPendingTransaction(event)
+  const res = await Wallets.addPendingTransaction(event)
+  if (res instanceof CouldNotFindWalletFromOnChainAddressError) {
```

Comment:
What happens to tracing here? Don't want this error to trigger an alert.

Reply:
It shouldn't trigger an alert, its level inherits from `DomainError` as `ErrorLevel.Info`.